.gitlab/kernel_matrix_testing/common.yml

```diff
@@ -268,3 +268,54 @@
     reports:
       annotations:
         - $EXTERNAL_LINKS_PATH
+
+notify_ebpf_complexity_changes:
+  stage: notify
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner$TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX:$TEST_INFRA_DEFINITIONS_BUILDIMAGES
+  tags: ["arch:amd64"]
+  rules:
+    - !reference [.on_system_probe_or_e2e_changes_or_manual]
+    - !reference [.on_security_agent_changes_or_manual]
+    - !reference [.except_mergequeue]
+    - !reference [.except_main_or_release_branch]
+    - !reference [.except_no_tests_no_deploy]
+  needs:
+    # We need to specify the jobs that generate complexity explicitly, else we hit the limit of "needs"
```

Comment:
Do you still hit the limit if you just specify `kmt_run_sysprobe_tests_x64` and not the individual parts of the matrix?

Reply:
If we want just the `kmt_run_sysprobe_tests_x64` we wouldn't, but then we'd lose the `arm64` data.